future-apps/payroll/contracts/Payroll.sol

```diff
@@ -384,8 +384,8 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
      * @return True if the given address is an active employee, false otherwise
      */
     function canForward(address _sender, bytes) public view returns (bool) {
-        // Check employee exists (and matches)
-        return (employees[employeeIds[_sender]].accountAddress == _sender);
+        // Check sender is active employee
+        return _isEmployeeActive(employeeIds[_sender]);
```

Comment:
Even though terminated employees would eventually be removed, and therefore fail the previous check, we should also disallow employees who have been terminated but may be lingering as they are technically not part of payroll anymore.

Reply:
I'm thinking of the other actions employees are allowed to perform even when they are terminated, like claiming their salaries. Could this still be useful in those cases when the calling employee is terminated?